src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,55 @@
+@import './variables';
+
+@mixin product-title-font {
+  font-family: $card-ff;
+  color: $dark-color;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+@mixin product-code-font-10 {
+  font-family: $card-ff;
+  color: $dark-color;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+@mixin product-code-font-12 {
+  font-family: $card-ff;
+  color: $grey-color;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+@mixin price-font {
+  font-family: $card-ff;
+  color: $dark-color;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+@mixin buy-button-font {
+  font-family: $card-ff;
+  color: $white-color;
+  background-color: $blue-color;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+}
+
+@mixin flex-center {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  box-sizing: border-box;
+}
+
+@mixin flex-left {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+}
```

Comment:
Mixin names should be in kebab-case. Rename `flex-left` to `flex-left` to comply with the code style rules.